NOTES-ANDROID.md

```diff
@@ -21,7 +21,7 @@ Notes for Android platforms
  (`*MIPS` targets are no longer supported with NDK R20+).
 
  Do not pass --cross-compile-prefix (as you might be tempted), as it
- will be "calculated" automatically based on chosen platform. However,
+ will be "calculated" automatically based on the chosen platform. However,
```

Comment:
We can improve this still further by replacing `"calculated"` with `determined` (removing the double quotes)